x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/fleet_event_filters_card.test.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import { I18nProvider } from '@kbn/i18n/react';
+import { FleetEventFiltersCard } from './fleet_event_filters_card';
+import * as reactTestingLibrary from '@testing-library/react';
+import { EventFiltersHttpService } from '../../../../../event_filters/service';
+import { useToasts } from '../../../../../../../common/lib/kibana';
+import { getMockTheme } from '../../../../../../../../public/common/lib/kibana/kibana_react.mock';
+import { GetExceptionSummaryResponse } from '../../../../../../../../common/endpoint/types';
+
+jest.mock('./exception_items_summary');
+jest.mock('../../../../../event_filters/service');
+
+jest.mock('../../../../../../../../../../../src/plugins/kibana_react/public', () => {
+  const originalModule = jest.requireActual(
+    '../../../../../../../../../../../src/plugins/kibana_react/public'
+  );
+  const useKibana = jest.fn().mockImplementation(() => ({
+    services: {
+      http: {},
+      data: {},
+      notifications: {},
+      application: {
+        getUrlForApp: jest.fn(),
+      },
+    },
+  }));
+
+  return {
+    ...originalModule,
+    useKibana,
+  };
+});
+
+jest.mock('../../../../../../../common/lib/kibana');
+
+const mockTheme = getMockTheme({
+  eui: {
+    paddingSizes: { m: '2' },
+  },
+});
+
+const EventFiltersHttpServiceMock = EventFiltersHttpService as jest.Mock;
+const useToastsMock = useToasts as jest.Mock;
+
+const summary: GetExceptionSummaryResponse = {
+  windows: 3,
+  linux: 2,
+  macos: 2,
+  total: 7,
+};
+
+describe('Fleet event filters card', () => {
+  let promise: Promise<GetExceptionSummaryResponse>;
+  let addDanger: jest.Mock = jest.fn();
+  const renderComponent: () => Promise<reactTestingLibrary.RenderResult> = async () => {
+    const Wrapper: React.FC = ({ children }) => (
+      <I18nProvider>
+        <ThemeProvider theme={mockTheme}>{children}</ThemeProvider>
+      </I18nProvider>
+    );
+    // @ts-ignore
+    const component = reactTestingLibrary.render(<FleetEventFiltersCard />, { wrapper: Wrapper });
+    try {
+      // @ts-ignore
+      await reactTestingLibrary.act(() => promise);
+    } catch (err) {
+      return component;
+    }
+    return component;
+  };
+  afterEach(() => {
```

Comment:
nitpick: semantically it makes sense to keep all `before...` methods before `after...` methods. So `beforeAll` followed by `beforeEach` and then `afterEach` at the end.